easybuild/easyconfigs/i/ITK-SNAP/ITK-SNAP-3.6.0-foss-2018a.eb

```diff
@@ -0,0 +1,27 @@
+# This easyconfig was created by the BEAR Software team at the University of Birmingham.
+easyblock = 'Tarball'
+name = 'ITK-SNAP'
+version = '3.6.0'
+
+homepage = "http://www.itksnap.org/pmwiki/pmwiki.php?n=Main.HomePage"
+description = """ITK-SNAP is a software application used to segment structures in 3D medical images.
+It is the product of a decade-long collaboration between Paul Yushkevich, Ph.D., of the Penn Image
+Computing and Science Laboratory (PICSL) at the University of Pennsylvania, and Guido Gerig, Ph.D.,
+of the Scientific Computing and Imaging Institute (SCI) at the University of Utah, whose vision was
+to create a tool that would be dedicated to a specific function, segmentation, and would be easy to
+use and learn. ITK-SNAP is free, open-source, and multi-platform."""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
```

Comment:
@edmondac Why use a non-`dummy` toolchain if nothing is actually getting compiled?

Also, we strongly prefer compiling from source if there's an option to.
Is there in this case? And if so, why did you take the binary road?

Reply:
Yes - I'd forgotten the details about this one. It just looked simple and I thought I'd try out --new-pr. Hmm.

I built this just after ANTs, and was fed up with compiling ITK and related things. Probably should just close this one without merging...